src/components/Header/Desktop.tsx

```diff
@@ -165,6 +165,16 @@ const OpenSourceLinks = () => {
           icon={['fad', 'database']}
           tag="Mocking"
         />
+
+        <ProductLink
+          name="Elements"
+          description="Web components for customizable and interactive API Documentation ."
+          className="ml-4"
+          color="green"
+          href="/open-source/elements"
+          icon={['fad', 'paint-brush']}
+          tag="Elements"
```

Comment:
```suggestion
          tag="Documentation"
```

Is it this which changes this?